rebasehelper/utils.py

```diff
@@ -115,27 +123,33 @@ def should_use_colors(cls, conf):
         return True
 
     @classmethod
-    def cprint(cls, message, color=None):
-        """
-        Print colored output if possible
+    def cprint(cls, message, fg=None, bg=None, style=None):
+        """Prints colored output if possible.
+
+        Args:
+            message (str): String to be printed out.
+            fg (str): Color to be used as a foreground in the printed message.
+            bg (str): Color to be used as a background in the printed message.
```

Comment:
This is unnecessarily verbose, just `foreground color`/`background color` is enough.